packages/nx/src/native/index.js

```diff
@@ -75,10 +75,21 @@ Module._load = function (request, parent, isMain) {
     if (existsSync(tmpFile)) {
       return originalLoad.apply(this, [tmpFile, parent, isMain]);
     }
-    if (!existsSync(nativeFileCacheLocation)) {
-      mkdirSync(nativeFileCacheLocation, { recursive: true });
+
+    mkdirSync(nativeFileCacheLocation, { recursive: true });
+
+    try {
+      copyFileSync(nativeLocation, tmpFile, constants.COPYFILE_EXCL);
+    } catch (e) {
+      if (e.code === 'EEXIST') {
```

Comment:
I tested this logic locally:

![image](https://github.com/user-attachments/assets/5b3baced-8269-4529-a275-a17c30f556e4)